CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+3.0.0 (2020-10-20)
```

Comment:
Cara, pra evitar conflito com o loafer do george era bom por uma tag aqui. Talvez `3.0.0+pre1`.
Vc vai subir essas modificações pro repo dele também, certo?

Reply:
então a principio esse cara iria subir só para olist loafer mesmo.

Reply:
E como q estamos criando o pacote no packagecloud? Ta indo como olist-loafer ou só loafer? Se for só loafer pode rolar conflito num futuro.

Reply:
@luizdepra está indo para o PyPI como `olist-loafer`: https://pypi.org/project/olist-loafer/

Reply:
Isso, por já estar indo como `olist-loafer` pro PyPI não teria problema

Reply:
Hoje nossa versão que usamos já não é igual a versão que esta na master do `loafer`

Reply:
Então tá suave. :)